conjure-codegen/src/context.rs

```diff
@@ -750,6 +752,50 @@ impl Context {
 
         Ident::new(&name, Span::call_site())
     }
+
+    pub fn module_path(&self, name: &TypeName) -> Vec<String> {
+        let raw = self.raw_module_path(name.package());
+
+        if raw.starts_with(&self.strip_prefix) {
+            raw[self.strip_prefix.len()..].to_vec()
+        } else {
+            raw
+        }
+    }
+
+    fn raw_module_path(&self, package: &str) -> Vec<String> {
+        package.split('.').map(|s| self.ident_name(s)).collect()
+    }
+
+    fn type_path(&self, this_type: &TypeName, other_type: &TypeName) -> TokenStream {
```

Comment:
Is there a test that covers the case of types referring to types that aren't direct parents/children?

Reply:
Oh good point, I'll add one covering sibling packages.

Reply:
Updated